Uppsala/UppEt3.xml

```diff
@@ -0,0 +1,557 @@
+                                <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">30</citedRange></bibl> notes
+                                that this text is written by the same hand as the <ref target="#ms_i2.20"/>.
+                            </note>
+                            <incipit xml:lang="gez">
+                                ኦእግዚኦ፡ ነአኵተከ፡ ወንሴብሐከ፡ ወንሰግድ፡ ለዕበየ፡ ሐይልከ፡ ወለጽንዓ፡ ፍቅርከ፡ እስመ፡ ፈኖከ፡ ለነ፡ ወልድከ፡ 
+                                <sic resp="PRS6337Lofgren">ፍቁርከ፡</sic> ወኢምሕኮ፡ እምኔነ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                ኢነጸረቶ፡ እሙ፡ በጊዜ፡ ሞቱ፡ ወኢኦድዎ፡ አዕርክቲሁ፡ ለሐውጾቱ። ጻመወ፡ በፈቃዱ፡ ባሕቲቱ፡ እስመ፡ መሐሪ፡ 
+                                ውእቱ፡ ወመፍቀሬ፡ ሰብእ፡ ይብትክ፡ ወይፍታሕ፡ መአስረ፡ ኀጣውኢነ፡ ኀበ፡ ሀሎ፡ ነሀሉ፡ በከመ፡ ለሊሁ፡ አሰፈወነ፡ 
+                                ሎቱ፡ ስብሐት፡ <add place="above">ዘ</add>እምኀቤነ፡ ወለነኒ፡ መንግሥቶ፡ ይክፍለነ፡ ይእዜኒ፡ ወዘልፈኒ፡ 
+                                ወለዓለመ፡ ዓለም፡ አሜን።
+                            </explicit>
+                        </msItem>
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">80</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>154</height>
+                                        <width>112</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="16 17">
+                                    <dimensions unit="mm" xml:id="textarea1">
+                                        <height>97</height>
+                                        <width>75</width>
+                                    </dimensions>
+                                    <note>Data on dimensions have been adopted from the entry on the manuscript in
+                                        <bibl><ptr target="bm:alvin"/></bibl>.
+                                    </note>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">31</citedRange>
+                                </bibl> calls it a "large and even archaic script" 
```

Comment:
Please note that this characteristic is in part misleading today (15th century handwriting would not be thought as "archaic", this would refer to pre-mid-14th century hands). The characteristic might be: "Regular, careful; by a good scribe. The letters are of big size" (the reference without numerical value is also a bit shaky).